src/Mvc/ViewCompilation/test/FunctionalTests/Infrastructure/TestEmbeddedResource.cs

```diff
@@ -24,7 +24,7 @@ public static void AssertContent(string resourceFile, string actual)
             {
                 var solutionRoot = ApplicationPaths.SolutionDirectory;
                 var projectName = typeof(TestEmbeddedResource).GetTypeInfo().Assembly.GetName().Name;
-                var fullPath = Path.Combine(solutionRoot, "test", ProjectName, "Resources", resourceFile);
+                var fullPath = Path.Combine(solutionRoot, "ViewCompilation", "test", "FunctionalTests", "Resources", resourceFile);
```

Comment:
Where does the `ProjectName` go?
Also, shouldn't it be `projectName` insetad - as the above variable is named?

Reply:
The assembly names no longer match up with the folder structure. Removed the variable